recipes/r-tangles/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set version = '0.8.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-tangles
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/tangles_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/tangles/tangles_{{ version }}.tar.gz
+  sha256: 7013b201d2aa22889f12c695783d076057fe43dcc08e2fd935f9ed098b854519
+
+build:
+  merge_build_host: True  # [win]
+  skip: True  # [win]
```

This seems to be a noarch package... It maybe cannot be installed on windows due to missing dependencies, but noarch packages are built+uploaded on linux.
```suggestion
  noarch: generic
```